sdk/include/ndk/sefuncs.h

```diff
@@ -165,6 +165,19 @@ NtAdjustPrivilegesToken(
     _Out_writes_bytes_to_opt_(BufferLength, *ReturnLength) PTOKEN_PRIVILEGES PreviousState,
     _When_(PreviousState != NULL, _Out_) PULONG ReturnLength);
 
+_Must_inspect_result_
+__kernel_entry
+NTSYSCALLAPI
+NTSTATUS
+NTAPI
+NtFilterToken(
```

Comment:
It appears (at least) the Nt*Token* functions are sorted alphabetically, so this one should certainly go after `NtDuplicateToken`.